src/udev/udev-builtin-path_id.c

```diff
@@ -660,7 +661,31 @@ static int builtin_path_id(sd_device *dev, sd_netlink **rtnl, int argc, char *ar
                                 path_prepend(&path, "cs-%s", sysnum);
                                 parent = skip_subsystem(parent, "spi");
                         }
-                }
+		} else if (streq(subsys, "nvme-subsystem")) {
```

Comment:
hmm, what's the difference between the subsystems called "nvme" and "nvme-subsystem"? this is weird...